lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,38 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden" tabindex="-1">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Your team could not created!") %></h3>
+            <span class="screen-reader-message sr"></span>
+            <div class="copy">
+                <p></p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="team-edit-fields">
+    <div class="team-instructions">
+        <p class="copy"><%- gettext("Enter information to describe your team. You cannot change these details after you create the team.") %></p>
+    </div>
+
+    <div class="team-required-fields">
+    </div>
+
+    <div class="team-optional-fields">
+    </div>
+</div>
+
+<div class="create-team form-actions">
+    <button class="action action-primary">
+        <%- gettext("Create") %>
```

Comment:
Furthermore, it does not make sense to me that the Edit Team View should render the link for submitting the form with the text "Create"; this view will also be used for editing existing topics, so it should probably say something like "Submit" by default. The text for this action should be parameterized so that whoever instantiates the Edit Team View can specify if they want it to say "Create" or "Submit".